Adds more presentation tips and strategies (#37)
vintasoftware/playbook: guidelines/guideline_presentations.md

    - [Pluggable Libs Through Design Patterns](https://www.youtube.com/watch?v=PfgEU3W0kyU)
    - [Scientific Hooliganism](https://www.youtube.com/watch?v=kOIVrWJqGJw)
    - [hunter2: A Symphony of Password Horror](https://www.youtube.com/watch?v=QLuMLoD4jzY)
- List of topics. You can just have a list of topics organized in a well thought sequence. Ideally, those are organized in way that things connect toghether.
    - [How to make a good library API PyCon 2017](https://www.youtube.com/watch?v=4mkFfce46zE)
    - [Critical Incidents: A Guide for Developers](https://www.youtube.com/watch?v=oQL4s01QbbU)
- Read the transcript. You can write down in slide notes the transcript of what you want to say and read those during the presentation. Be aware that requires **A LOT** of previous preparation for it to seem natural.
    - [It's about time by Russell Keith-Magee](https://www.youtube.com/watch?v=y13thlpbri8)
    - [ORM: The Sequel by Katie McLaughlin](https://www.youtube.com/watch?v=p9Bgr5VOISo)

## After Presentation Questions Tips

- If you want you can tell the room manager to that you will not be taking questions after the presentation.

> - If someone asks as question that you did not understand it's fine to ask them to re-state the question.

Reviewer: ```suggestion
- If someone asks a question that you did not understand, it's fine to ask them to restate the question.
```
